app/Livewire/GlobalSearch.php

```
@@ -419,22 +599,728 @@ private function loadSearchableItems()
+    {
+        $this->selectedServerId = $serverId;
+
+        if ($shouldProgress) {
+            $this->loadDestinations();
+        }
+    }
+
+    public function loadDestinations()
+    {
+        $this->loadingDestinations = true;
+        $server = Server::find($this->selectedServerId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method 'loadDestinations'. (undefined)

(StaticAccess)